modules/snooker/src/main/ts/ephox/snooker/api/ResizeBehaviour.ts

```diff
@@ -8,6 +11,7 @@ export interface ResizeBehaviour {
   readonly calcLeftEdgeDeltas: (sizes: number[], index: number, nextIndex: number, delta: number, minCellSize: number, relativeSizing: boolean) => number[];
   readonly calcMiddleDeltas: (sizes: number[], previousIndex: number, index: number, nextIndex: number, delta: number, minCellSize: number, relativeSizing: boolean) => number[];
   readonly calcRightEdgeDeltas: (sizes: number[], previousIndex: number, index: number, delta: number, minCellSize: number, relativeSizing: boolean) => number[];
+  readonly getNewWidths: (table: SugarElement<HTMLTableElement>, warehouse: Warehouse, tableSize: TableSize, pixelDelta: number) => { delta: number; newSizes: number[] };
```

Comment:
Maybe `calcRedestributedWidths`? As it's not really clear what "newWidths" is here.

Reply:
Also I still really think we should change this to be the same as the other APIs so it would be:
```
readonly getNewWidths: (sizes: number[], total: number, pixelDelta: number) => { delta: number; newSizes: number[] };
```
And then move getting the other details to where this is called in `Adjustments`.